enviro/compute/fitting.py

```diff
@@ -29,6 +29,9 @@ def _f2(x, a, b, c):
 def _f1(x, a, b, c):
     return a + b * x ** c
 
+# bounds for function parameters
+_bounds = ([np.finfo(np.float64).tiny, np.finfo(np.float64).tiny, -np.inf],
+          [np.inf, np.inf, np.inf])
```

Comment:
Please comment which value corresponds to which variable (a,b,c)

Reply:
There are comments now like the description of this pull request.